### PR TITLE
Fixing interquestion glitch

### PR DESCRIPTION
**Pull request by alanlanglois**
## Detailed purpose of the PR
Removing the glitch we had before the interquestion screen was displayed. 
(The next question was displayed quickly just before the correction). 
A place holder is displayed instead until the question is validated. 
| **Before** | **After** |
| ------------- | ------------- |
|  <image src="https://user-images.githubusercontent.com/712986/85396091-6cc43d80-b551-11ea-8879-546e37ddb6b7.gif" width="200"/> | <image src="https://user-images.githubusercontent.com/712986/85395171-01c63700-b550-11ea-8bb5-48bd1ba5726b.gif" width="200"/> |

### Testing strategy

- [x] I ensured my code is tested: test-unit / lint / flow / android emulator / real device pixel 3a
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/screens/question.js [hunk 1]
```diff
@@ -55,11 +55,25 @@ type Props = {|
   ...ConnectedDispatchProps
 |};
 
-class QuestionScreen extends React.PureComponent<Props> {
+type State = {|
+  isValidating: boolean
+|};
+
+class QuestionScreen extends React.Component<Props> {
```
Comment (Fausto95) [inline src/screens/question.js:62]:
```suggestion
class QuestionScreen extends React.Component<Props, State> {
```

Comment (alanlanglois) [reply src/screens/question.js]: fixed!

File: src/redux/actions/ui/answers.test.js [hunk 2]
```diff
@@ -29,41 +29,64 @@ const state = createStoreState({
...
       const {validateAnswer: _validateAnswer} = require('@coorpacademy/player-store');
 
-      const {validateAnswer} = require('./answers');
+      const answers = require('./answers');
 
       const dispatch = jest.fn();
       const getState = jest.fn(() => state);
 
+      // answers.changeAnswerValidationStatus = jest.fn();
+
```
Comment (Fausto95) [inline src/redux/actions/ui/answers.test.js:81]:
```suggestion
```

File: src/redux/reducers/ui/answers.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// @flow strict
+
+import {VALIDATE_ANSWER} from '../../actions/ui/answers';
+// import type {Action} from '../../actions/ui/answers';
```
Comment (Fausto95) [inline src/redux/reducers/ui/answers.test.js:4]:
```suggestion
```

**Comment by Fausto95** [comment]
[App | Lorsque je réponds, la question suivante s'affiche un court instant avant d'afficher les pops-in](https://trello.com/c/oxAeFbqZ/1765-app-lorsque-je-r%C3%A9ponds-la-question-suivante-saffiche-un-court-instant-avant-dafficher-les-pops-in)

**Pull request merged**